controllers/submariner/submariner_controller.go

```diff
@@ -227,6 +231,8 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, err
 	}
 
+	// TODO: vthapar Do we need proxy status?
```

Comment:
Seems that for consistency we should, no?

Reply:
Status bits are currently added to SUbmariner CR's status field. This will require additions to Submariner CR in `submariner` repo. We're also missing status for NetworkPluginSyncer. Should we bring them both in a separate issue?

Reply:
Yup, we shouldn’t take care of the syncer here, but I agree we should add the metrics status information here while we’re at it.

Reply:
@skitt @mkolesnik Code here is to update Submariner CR's Status fields with DaemonSet's status. Till we add the CR fields in `submariner` we can't do anything here.